### ISSUE TITLE
PHP support release date

### ISSUE THREAD

**Issue by antton** [baidu/openrasp#103]
Hi, 

I'd like to know if you have any release date supporting PHP, as I saw in a wiki the release would be soon but I also see that is going to work with php 5.2-5.6 what about 7? Is it in your roadmap?

Thanks!!

**Comment by CaledoniaProject** [comment]
Support PHP 5.X should be ready in late March, we're currently on Spring Festival so I don't have an exact release date.

A various kind of tests need to be done before it goes public, I'll let you know when it's ready.

And finally, PHP 7.X is on our roadmap. But we're going to do Java & PHP 5.X agent first, add optional remote management capability, and a control panel.

I'll add localization support (i18n) in the next release, but the documents would remain in Chinese for a while.

**Comment by antton** [comment]
Thanks!!
I'll wait until march and then try to collaborate with it!!

**Comment by CaledoniaProject** [comment]
Support PHP 5.X is ready. Please kindly refer to https://github.com/baidu/openrasp/releases

**Comment by CaledoniaProject** [comment]
[OpenRASP v0.40](https://github.com/baidu/openrasp/releases/tag/v0.40) is out, support of PHP 7.0~7.2 is included in this release.

Please open a new issue if there's any problem, closing this one.
